front/src/modules/simulationResult/components/ChartHelpers/drawPowerRestriction.ts

```diff
@@ -152,4 +154,149 @@ const drawPowerRestriction = (
     });
 };
 
+export const drawPowerRestrictionV2 = (
```

Comment:
If you have adapted getAxisValuesV2, you can remove this function
```suggestion
```

Reply:
Done.